src/CMakeLists.txt

```diff
@@ -1,17 +1,37 @@
 set (CMAKE_CXX_STANDARD 11)
 set (CMAKE_INTERPROCEDURAL_OPTIMIZATION TRUE)
 
+if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+  set (INSTALL_LOCAL TRUE)
+  message ("installing locally")
+else (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+  message ("installing to ${CMAKE_INSTALL_PREFIX}")
+endif (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+
 add_executable (jigsaw-cmd jigsaw.cpp)
 target_compile_definitions (jigsaw-cmd PRIVATE __cmd_jigsaw)
 set_target_properties (jigsaw-cmd PROPERTIES OUTPUT_NAME jigsaw)
-install (TARGETS jigsaw-cmd DESTINATION "${PROJECT_SOURCE_DIR}/bin")
+if (DEFINED INSTALL_LOCAL)
+  install (TARGETS jigsaw-cmd DESTINATION "${PROJECT_SOURCE_DIR}/bin")
+else ()
+  install (TARGETS jigsaw-cmd DESTINATION bin)
```

Comment:
@dengwirda, on conda-forge it doesn't work to install locally to the source tree.  You have to install to the `${CMAKE_INSTALL_PREFIX}` in order for the executables and libraries to become part of the package.  The easiest way I can figure out to do this is to see whether a `-DCMAKE_INSTALL_PREFIX` has been passed to `cmake` (or if it's still the default).  From the conda-forge recipe, I call:
```
cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_BUILD_TYPE=BUILD_MODE ..
```
whereas your recommended recipe excludes the first flag, and thus should still build locally.